custom-words.txt

```diff
@@ -171,6 +171,7 @@ appid
 appinsights
 Applicationkey
 applynetworkconfigurationupdates
+approverollingupgrade
```

Comment:
can you remove this and use camel case approveRollingUpgrade instead?

Reply:
(Also I didn't happen to notice the approveRollingUpgrade in the diff for this PR, did I overlook it?)

Reply:
https://github.com/Azure/azure-rest-api-specs/commit/5ba8d37a97046fa9b8b3d33757b6f28d370533ec#diff-22bcf7eb889078cf4dddb597d13d787092269075bddc646987b3504b694bcb1eR1761
found it here

Reply:
updated in the latest commit as advised